README.md

```diff
@@ -1 +1,34 @@
-# qe-sprayproxy
\ No newline at end of file
+# Spray Proxy Deployment
+
+This doc is to guide you how to use the scirpts to deploy Spray Proxy on a perment QE clsuter, which is used to distribute requests sent from GH App
+
```

Comment:
I'd add a **Prerequisites** section here with these bullet points:
* make sure you have permissions to manage RHTAP QE Github App https://github.com/organizations/redhat-appstudio-qe/settings/apps/rhtap-qe-app Github App
* make sure you have write permissions for https://vault.ci.openshift.org/ui/vault/secrets/kv/show/selfservice/redhat-appstudio-qe/ci-secrets

Reply:
totally agree, thank you